<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchProgressWidget</class>
 <widget class="QWidget" name="BatchProgressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch progress widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../main/piaf.qrc">
    <normaloff>:/images/22x22/system-run.png</normaloff>:/images/22x22/system-run.png</iconset>
  </property>
  <property name="toolTip">
   <string>Batch progress for one batch task</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="4" column="2" rowspan="3">
    <widget class="ImageWidget" name="imageLabel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" rowspan="3">
    <widget class="QFrame" name="frame_fileList">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="filesButtonsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="toolTip">
            <string>Add new files</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../main/piaf.qrc">
             <normaloff>:/images/22x22/list-add.png</normaloff>:/images/22x22/list-add.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+O</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delButton">
           <property name="toolTip">
            <string>Remove selected files</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../main/piaf.qrc">
             <normaloff>:/images/22x22/list-remove.png</normaloff>:/images/22x22/list-remove.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="toolTip">
            <string>Reset state to unprocessed</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../main/piaf.qrc">
             <normaloff>:/images/22x22/edit-undo.png</normaloff>:/images/22x22/edit-undo.png</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+Z</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QTreeWidget" name="filesTreeWidget">
        <property name="toolTip">
         <string>Files for batch processing</string>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="autoExpandDelay">
         <number>1</number>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>File</string>
         </property>
         <property name="icon">
          <iconset resource="../../../main/piaf.qrc">
           <normaloff>:/images/pixmaps/movie.png</normaloff>:/images/pixmaps/movie.png</iconset>
         </property>
        </column>
        <column>
         <property name="text">
          <string>State</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sequence:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="sequenceLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>--</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="greyButton">
        <property name="toolTip">
         <string>if down, the images will be converted to grayscale before processing</string>
        </property>
        <property name="statusTip">
         <string>Process input files in color or in grayscale: conversion is done before sending the images to plugins</string>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
        <property name="icon">
         <iconset resource="../../../main/piaf.qrc">
          <normaloff>:/images/22x22/view-color.png</normaloff>
          <activeon>:/images/22x22/view-gray.png</activeon>:/images/22x22/view-color.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="loadButton">
        <property name="toolTip">
         <string>Load plugin sequence</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../main/piaf.qrc">
          <normaloff>:/images/16x16/document-open.png</normaloff>:/images/16x16/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QCheckBox" name="reloadPluginCheckBox">
        <property name="toolTip">
         <string>Reload plugins at every change of file</string>
        </property>
        <property name="statusTip">
         <string>Reload the plugins at every change of file. If unchecked, all the images will be processed by the same plugin processus (beware of image size changes).</string>
        </property>
        <property name="text">
         <string>1 plug/file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_options" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QPushButton" name="recordButton">
        <property name="toolTip">
         <string>Record output image or movies</string>
        </property>
        <property name="statusTip">
         <string>Record output image when inputs are still images or movies when inputs are movies</string>
        </property>
        <property name="whatsThis">
         <string>Record output image or movies</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../main/piaf.qrc">
          <normaloff>:/images/22x22/media-record.png</normaloff>:/images/22x22/media-record.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="viewButton">
        <property name="toolTip">
         <string>Display the image output once per second</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../main/piaf.qrc">
          <normaloff>:/images/16x16/layer-visible-on.png</normaloff>:/images/16x16/layer-visible-on.png</iconset>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="snapButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../main/piaf.qrc">
          <normaloff>:/images/22x22/document-save.png</normaloff>:/images/22x22/document-save.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="TimeHistogramWidget" name="timeHistoWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Processing time histogram</string>
     </property>
     <property name="statusTip">
      <string>Processing time histogram</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(172, 172, 172);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QWidget" name="widget_progress" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_files" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Files:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="nbFilesLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Number of files in process list</string>
           </property>
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Proc:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="nbProcLabel">
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Err:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="nbErrLabel">
           <property name="text">
            <string>--</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QProgressBar" name="progressBar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="playPauseButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Start processing or pause it</string>
           </property>
           <property name="statusTip">
            <string>Start processing or pause it</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../ema.qrc">
             <normaloff>:/icons/22x22/media-playback-start.png</normaloff>
             <disabledoff>:/icons/22x22/media-playback-pause.png</disabledoff>:/icons/22x22/media-playback-start.png</iconset>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeHistogramWidget</class>
   <extends>QWidget</extends>
   <header>timehistogramwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ema.qrc"/>
  <include location="../../../main/piaf.qrc"/>
 </resources>
 <connections/>
</ui>
